project/BuildPlugin.scala

```diff
@@ -264,7 +264,7 @@ object BuildImplementation {
     BuildKeys.schemaVersion := "3.2",
     Keys.testOptions in Test += sbt.Tests.Argument("-oD"),
     Keys.onLoadMessage := Header.intro,
-    Keys.onLoad := BuildDefaults.bloopOnLoad.value,
+    //Keys.onLoad := BuildDefaults.bloopOnLoad.value,
```

Comment:
It looks like this should be enabled back :wink:

Reply:
Yes I remember that yester when I was in the subway. I had already amended that this morning. It's strange you still see that in the diff.